src/r_exasol/connection/connection_factory_impl.cpp

```diff
@@ -23,8 +23,6 @@ std::shared_ptr<exa::ConnectionEstablisher> exa::ConnectionFactoryImpl::createCo
         case https:
             conn_est = std::make_shared<HttpsConnectionEstablisher>(getCertificate());
             break;
-        default:
```

Comment:
I guess this is necessary, because package on cran are not allowed to kill the R process. Should we log it instead? Can it happen.

Reply:
I added a log, although it is very unlikely that it runs actuallly  ever into the default case.